### ISSUE TITLE
Netlify build error

### ISSUE THREAD

**Issue by xplosionmind** [manunamz/jekyll-wikilinks#13]
I am experiencing an issue only while building with Netlify. Any ideas about why could the problem be? Locally, everything is working fine.

deploy log (full log [here](https://app.netlify.com/teams/xplosionmind/builds/60c489749d8db3c95f323e52)):
```
10:19:17 AM: /opt/build/cache/bundle/ruby/3.0.0/gems/jekyll-wikilinks-0.0.3/lib/jekyll-wikilinks.rb:177:in `write': No such file or directory @ rb_sysopen - /opt/build/repo/assets/graph-net-web.json (Errno::ENOENT)
10:19:17 AM: 	from /opt/build/cache/bundle/ruby/3.0.0/gems/jekyll-wikilinks-0.0.3/lib/jekyll-wikilinks.rb:177:in `write_graph_data'
	from /opt/build/cache/bundle/ruby/3.0.0/gems/jekyll-wikilinks-0.0.3/lib/jekyll-wikilinks.rb:58:in `generate'
	from /opt/build/cache/bundle/ruby/3.0.0/gems/jekyll-4.2.0/lib/jekyll/site.rb:193:in `block in generate'
	from /opt/build/cache/bundle/ruby/3.0.0/gems/jekyll-4.2.0/lib/jekyll/site.rb:191:in `each'
	from /opt/build/cache/bundle/ruby/3.0.0/gems/jekyll-4.2.0/lib/jekyll/site.rb:191:in `generate'
	from /opt/build/cache/bundle/ruby/3.0.0/gems/jekyll-4.2.0/lib/jekyll/profiler.rb:43:in `block in profile_process'
	from /opt/build/cache/bundle/ruby/3.0.0/gems/jekyll-4.2.0/lib/jekyll/profiler.rb:41:in `each'
	from /opt/build/cache/bundle/ruby/3.0.0/gems/jekyll-4.2.0/lib/jekyll/profiler.rb:41:in `profile_process'
	from /opt/build/cache/bundle/ruby/3.0.0/gems/jekyll-4.2.0/lib/jekyll/site.rb:75:in `process'
	from /opt/build/cache/bundle/ruby/3.0.0/gems/jekyll-4.2.0/lib/jekyll/command.rb:28:in `process_site'
	from /opt/build/cache/bundle/ruby/3.0.0/gems/jekyll-4.2.0/lib/jekyll/commands/build.rb:65:in `build'
	from /opt/build/cache/bundle/ruby/3.0.0/gems/jekyll-4.2.0/lib/jekyll/commands/build.rb:36:in `process'
	from /opt/build/cache/bundle/ruby/3.0.0/gems/jekyll-4.2.0/lib/jekyll/command.rb:91:in `block in process_with_graceful_fail'
	from /opt/build/cache/bundle/ruby/3.0.0/gems/jekyll-4.2.0/lib/jekyll/command.rb:91:in `each'
	from /opt/build/cache/bundle/ruby/3.0.0/gems/jekyll-4.2.0/lib/jekyll/command.rb:91:in `process_with_graceful_fail'
	from /opt/build/cache/bundle/ruby/3.0.0/gems/jekyll-4.2.0/lib/jekyll/commands/build.rb:18:in `block (2 levels) in init_with_program'
	from /opt/build/cache/bundle/ruby/3.0.0/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `block in execute'
	from /opt/build/cache/bundle/ruby/3.0.0/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `each'
	from /opt/build/cache/bundle/ruby/3.0.0/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `execute'
	from /opt/build/cache/bundle/ruby/3.0.0/gems/mercenary-0.4.0/lib/mercenary/program.rb:44:in `go'
	from /opt/build/cache/bundle/ruby/3.0.0/gems/mercenary-0.4.0/lib/mercenary.rb:21:in `program'
	from /opt/build/cache/bundle/ruby/3.0.0/gems/jekyll-4.2.0/exe/jekyll:15:in `<top (required)>'
	from /opt/build/cache/binstubs/jekyll:29:in `load'
	from /opt/build/cache/binstubs/jekyll:29:in `<main>'
```

**Comment by manunamz** [comment]
I would guess it has to do with the `/assets` directory again. If you are doing automatic deploys directly from github, there very likely is no `/assets` directory because git doesn't save empty directories. Therefore, when netfliy goes to write the graph data, the required directory is missing, and then the build chokes.

So, a hacky workaround would be to add a file in the `/assets` directory to your repo, perhaps a file like `/assets/blank.txt` just to make sure the directory is there when the graph data needs to be written. 

I do not recommend this as a permanent or production solution. I am definitely going to change this behavior soon. BUT the above hack should get your build working immediately. :)

**Comment by xplosionmind** [comment]
> I am definitely going to change this behavior soon.

Thank you very much.  
I will now close this issue since it has the same fix of #11.
